app/Jobs/WikiEntityImportJob.php

```diff
@@ -0,0 +1,240 @@
+            );
+        }
+        return $jobName;
+    }
+
+    private function constructSpec(): array
+    {
+        return [
+            'metadata' => [
+                'generateName' => 'run-transferbot-',
+                'namespace' => $this->kubernetesNamespace,
+                'labels' => [
+                    'app.kubernetes.io/instance' => $this->wiki->domain,
+                    'app.kubernetes.io/name' => 'run-transferbot',
+                ]
+            ],
+            'spec' => [
+                'ttlSecondsAfterFinished' => 0,
+                'backoffLimit' => 0,
+                'template' => [
+                    'metadata' => [
+                        'name' => 'run-transferbot'
+                    ],
+                    'spec' => [
+                        'containers' => [
+                            0 => [
+                                'hostNetwork' => true,
+                                'name' => 'run-qs-updater',
+                                'image' => 'ghcr.io/wbstack/transferbot:main',
+                                'env' => [
+                                    ...$this->creds->marshalEnv(),
+                                    [
+                                        'name' => 'CALLBACK_ON_FAILURE',
+                                        'value' => 'curl -H "Accept: application/json" -H "Content-Type: application/json" --data \'{"wiki_entity_import":'.$this->importId.',"status":"failed"}\' -XPATCH http://api-app-backend.default.svc.cluster.local/backend/wiki/updateEntityImport'
+                                    ],
+                                    [
+                                        'name' => 'CALLBACK_ON_SUCCESS',
+                                        'value' => 'curl -H "Accept: application/json" -H "Content-Type: application/json" --data \'{"wiki_entity_import":'.$this->importId.',"status":"success"}\' -XPATCH http://api-app-backend.default.svc.cluster.local/backend/wiki/updateEntityImport'
+                                    ],
+                                ],
+                                'command' => [
+                                    'transferbot',
+                                    $this->sourceWikiUrl,
+                                    $this->targetWikiUrl,
+                                    ...$this->entityIds,
+                                ],
+                                'resources' => [
+                                    'requests' => [
+                                        'cpu' => '0.25',
+                                        'memory' => '250Mi',
+                                    ],
+                                    'limits' => [
+                                        'cpu' => '0.5',
+                                        'memory' => '500Mi',
+                                    ],
+                                ],
```

Comment:
This is not even an educated guess. I would like to see this run in staging against whatever we decide to import before coming up with real world numbers.

Reply:
I think generally keeping the request low and the limit high here will be fine so it can burst to whatever is available. We con't really need this thing to run "realtime" like stuff that's replying to webrequests but some real world numbers will probably make us happy